docs/what-is-rocm.rst

```diff
@@ -99,14 +99,14 @@ Tools
 
   ":doc:`AMD SMI <amdsmi:index>`", "C library for Linux that provides a user space interface for applications to monitor and control AMD devices"
   ":doc:`HIPIFY <hipify:index>`", "Translates CUDA source code into portable HIP C++"
-  "`RocBandwidthTest <https://github.com/ROCm/rocm_bandwidth_test/>`_ ", "Captures the performance characteristics of buffer copying and kernel read/write operations"
+  "`RocBandwidthTest <https://rocm.docs.amd.com/projects/rocm_bandwidth_test/en/latest/>`_", "Captures the performance characteristics of buffer copying and kernel read/write operations"
   ":doc:`ROCmCC <./reference/rocmcc>`", "Clang/LLVM-based compiler"
   ":doc:`ROCm CMake <rocmcmakebuildtools:index>`", "Collection of CMake modules for common build and development tasks"
   ":doc:`ROCm Data Center Tool <rdc:index>`", "Simplifies administration and addresses key infrastructure challenges in AMD GPUs in cluster and data-center environments"
   "`ROCm Debug Agent (ROCdebug-agent) <https://github.com/ROCm/rocr_debug_agent/>`_ ", "Prints the state of all AMD GPU wavefronts that caused a queue error by sending a SIGQUIT signal to the process while the program is running"
   ":doc:`ROCm Debugger (ROCgdb) <rocgdb:index>`", "Source-level debugger for Linux, based on the GNU Debugger (GDB)"
   ":doc:`ROCdbgapi <rocdbgapi:index>`", "ROCm debugger API library"
-  "`rocminfo <https://github.com/ROCm/rocminfo/>`_ ", "Reports system information"
+  "`rocminfo <https://rocm.docs.amd.com/projects/rocminfo/en/latest/>`_", "Reports system information"
```

Comment:
```suggestion
  "`ROCm Info <https://rocm.docs.amd.com/projects/rocminfo/en/latest/>`_", "Reports system information"
```

Reply:
no spaces is correct

rocminfo is the most common spelling I've seen since that is also the name of the tool, but their docs use "ROCmInfo"
https://rocm.docs.amd.com/projects/rocminfo/en/latest/

Reply:
@Danny213123 please take a look at this PR and include in your proposal's on naming